src/test/java/GoRestTest.java

```diff
@@ -0,0 +1,77 @@
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class GoRestTest {
+
+    @BeforeClass
+    public void setUp() {
+        RestAssured.baseURI = "https://gorest.co.in/";
+        RestAssured.basePath = "public-api/";
+    }
+
+    @Test
+    public void testGoRest() {
+        final String token = "a27efba58da81f96b92922bc4ed805103264a09f6819ac931005cd609f7419fa";
+
+        final String body = """
+                    {
+                    "name": "Shake",
+                    "email": "ShakeGh_11@epam.com",
```

Comment:
I suggest having more dynamic body, using e.g. timestamp after name/email

Reply:
Addressed in e7a4f19.